src/Tools/IiifUrlValidator.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
I know there is a more Drupal-y way to write this. I wrote code in a Constraint that I didn't push. I would like to ask you about the best approach in Drupal!

Reply:
Ok. I think this is Ok the way it is. Maybe we can move this class  to the SBF module afterwards so we can use the same validation for the general IIIF form. I Would not do a Constraint here since this is not a "data type", means you are not using the type data api. Also, its not only about the shape but its calling a service (http). Are you only returning the message? Maybe it would be easier to return some constants and let the form/caller deal with the actual error reporting? Not a necessity. I like how its done

Reply:
Re: using on the general IIIF form, you don't mean `src/Form/IiifSettingsForm.php` in format_strawberryfield? Because I am already using the validator there, as well as in my new base formatter. Just want to clarify.

Reply:
MAN! Totally right. I forgot we moved that form into this module a few months ago. Sorry!